plugins/xivo-yealink/v84/plugin-info

```diff
@@ -134,12 +134,29 @@
             "expansion_modules": 0,
             "protocol": "sip"
         },
-        "Yealink, T57W, 58.84.0.30": {
+        "Yealink, T57W, 97.84.0.85": {
             "lines": 16,
             "high_availability": false,
             "function_keys": 0,
             "expansion_modules": 0,
             "protocol": "sip"
+ 	},
+        "Yealink, W52P, 25.81.0.30": {
```

Comment:
We should not have devices using firmware version 81 in the plugin made for version 84. For your information, the version number is the second number in the version string, i.e 25.**81**.0.30
Knowing that, I see that the W60B is only available in version 83, so it should be in the plugin version 83.

Reply:
Thanks for your reply. I didn't understand firmware version corresponded with plugin version.
Wouldn't it be simpler to have all equipments in one plugin ?

Reply:
No, as there may be differences depending on the firmware version :smile:. A provisioning plugin is written specifically for a firmware version. The plugin version is not the same. It is used to be able to update the plugin if changes are made.